pkgs/development/tools/vim-vint/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, pkgs, python3Packages, stdenv }:
+
+python3Packages.buildPythonApplication rec {
+  name = "vim-vint-${version}";
+  version = "0.3.11";
+
+  src = pkgs.fetchurl{
+    url = "mirror://pypi/v/vim-vint/${name}.tar.gz";
+    sha256 = "cda7090116b3aea12bdfe8ed44e71ead739df4e7e0ad24e27bb87754dedb613b";
+  };
+
+  # OSX keeps getting error: `Download error on https://pypi.python.org/simple/coverage: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:720) -- Some packages may not be found!`
```

Comment:
`pip` automatically attempts to download dependencies but we've blocked that. You should provide the dependencies via nix.